### DISCUSSION TITLE
How do I enable notifications for the discussions on the org/user level?

### DISCUSSION THREAD

**Discussion by ascheman** [community/community#73201] [Discussions] [answered]
### Select Topic Area

Question

### Body

Perhaps this is an FAQ? 

I couldn't figure out how to enable notifications on discussions on the org/user level. I follow an organization (i.e., https://github.com/konveyor). Does this mean I will get notified about new discussion items? How could I enable this otherwise? Would it be possible to restrict notifications to single categories?

Thanks in advance!

**Comment by TAIHAN86** [comment]
response 1

**Reply by ascheman** [reply]
How can I mark responses as Spam?

**Reply by mdavic0** [reply, marked as answer]
Marking responses as spam on GitHub discussions is not a standard feature provided by GitHub. However, if you come across spammy or inappropriate content in a discussion, you can take the following actions:

Report the user or content: You can report the user or specific content by clicking the 'Report abuse' or 'Report' button, typically found next to the user's comment or content. This will notify GitHub's moderation team, and they will review the reported content for any violations of GitHub's community guidelines.

Contact the organization or repository owner: If the spam is within a specific organization or repository, you can contact the organization or repository owner, and they can take action on the content or user as necessary.

Remember to be respectful and accurate when reporting content. GitHub's community guidelines are in place to maintain a positive and productive environment for all users.

If you have further questions or need additional assistance, please don't hesitate to ask.

**Comment by mdavic0** [comment]
Enabling notifications for discussions within an organization or user level is not as straightforward as following repositories. By simply following an organization, you won't automatically receive notifications for new discussion items.

To get notified about new discussions within an organization, you can:

Watch the organization: When you follow an organization, you'll receive notifications for new repositories, but not for discussions. To get discussion notifications, you can click the 'Watch' button on the organization's main page.

Manage notification settings: You can customize your notification preferences by going to your GitHub settings. Under 'Notifications,' you can choose which types of notifications you want to receive, and you can select specific repositories or categories to receive notifications for. However, at the time of my last update in January 2022, GitHub didn't provide granular control for discussions within organizations.

Use a GitHub Action or bot: You can also create custom solutions using GitHub Actions or bots to monitor discussions and notify you based on specific criteria. This requires some programming knowledge.

GitHub's features and options may have evolved since my last update, so I recommend checking GitHub's official documentation or the latest GitHub help resources for the most up-to-date information on managing notifications and discussions.

I hope this helps, and if you have more questions, feel free to ask

**Reply by ascheman** [reply]
Thanks for the extensive reply, @mdavic0. Please take a look at the [konveyor](https://github.com/konveyor) organization page. Where can I find the watch button?

**Reply by mdavic0** [reply]
Oh, I apologize! Now I see your problem; I misunderstood you. The 'Watch' option is only for specific repositories. I've found that enabling notifications for discussions is a known issue: [community/discussions/3951](https://github.com/orgs/community/discussions/3951). Also here: [integrations/slack/issues/1026](https://github.com/integrations/slack/issues/1026)

For the time being, I found that there are some integrations to get notifications using webhooks: [webhook-events-and-payloads#discussion](https://docs.github.com/en/webhooks/webhook-events-and-payloads#discussion). For example, here you can find a discussion notification integrated with Slack, but it's still in beta and doesn't have full support for discussion notifications; it only covers discussions created or answered: [integrations/slack#subscribe-to-an-organization-or-a-repository](https://github.com/integrations/slack#subscribe-to-an-organization-or-a-repository).

I hope this helps!
